src/main/scala/com/databricks/labs/overwatch/MultiWorkspaceDeployment.scala

```diff
@@ -313,17 +314,43 @@ class MultiWorkspaceDeployment extends SparkSessionWrapper {
     true
   }
 
+
+  private def generateBaseConfig(configCsvPath: String): DataFrame = {
```

Comment:
should we change param name from configCSVPath to configPath or something since it's no longer required to be a csv? Wondering if we should just go ahead and make the switch in the deployment runner now too -- easier to change now than later.

Reply:
done